packages/cozy-clisk/package.json

```diff
@@ -23,16 +23,18 @@
   "homepage": "https://github.com/konnectors/libs#readme",
   "scripts": {
     "lint": "eslint 'src/**/*.js'",
+    "build:types" : "yarn tsc -p tsconfig.json",
     "prepublishOnly": "yarn run build",
-    "build": "babel src/ -d dist/ --copy-files --verbose --ignore '**/*.spec.js','**/*.spec.jsx'",
+    "build": "yarn build:types && babel --extensions .ts,.js src/ -d dist/ --copy-files --verbose --ignore '**/*.spec.js','**/*.spec.ts'",
```

Comment:
Do we have a check in CI that ensure commited types are aligned with current code?

Reply:
I'll try, but for me, since yarn build:types will return an error code, the pipeline will fail by itself